_includes/head.html

```diff
@@ -3,16 +3,16 @@
 {% endif %}
 
 <head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
-  <meta name="description" content="Write an awesome description for your new site here. You can edit this line in _config.yml. It will appear in your document head meta (for Google search resu...">
+	<title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
+	<meta name="description" content="Write an awesome description for your new site here. You can edit this line in _config.yml. It will appear in your document head meta (for Google search resu...">
 
-  <link rel="stylesheet" href="{{site.baseurl}}/assets/main.css">
-  <link rel="stylesheet" href="{{site.baseurl}}/assets/site.css">
-  <link rel="alternate" type="application/rss+xml" title="SMC Style Guide" href="/feed.xml">
-  
-  
+	<link rel="stylesheet" href="{{site.baseurl}}/assets/main.css">
+	<link rel="alternate" type="application/rss+xml" title="SMC Style Guide" href="/feed.xml">
+
+	<script src="https://use.typekit.net/xzs4qjr.js"></script>
+	<script>try{Typekit.load({ async: true });}catch(e){}</script>
 </head>
```

Comment:
Make formatting generally and indentation generally part of the style guide.

What is our rule, team, for HTML? 2 or 4 spaces?

Personally, I favor 2. But I don't care so long as it is consistent.

This looks like 6-space indentation, which I am definitely against.

@waidelacruz 
@mandythomas 
@mirabelrice 
@kaylaziegler